src/pages/journal/components/journal-early-start-modal/journal-early-start-modal.scss

```diff
@@ -0,0 +1,19 @@
+journal-early-start-modal {
+  @extend .modal-alert-wrapper;
+
+  .modal-alert {
+    width: 500px;
```

Comment:
where did you get this number from? :)

Reply:
Honestly, its just a number which I had concocted myself as it didn't seem standardised when looking through the other modals, so went with something that made the text appear nicely wrapped - is there a specific value it should be set to?

Reply:
There must be a specific value, I'm not sure what that is though. Oli or Nad will be able to tell you

Reply:
We have reached out to Oli, updated this value to 450px, changed the Cancel button text to 'Return to journal' and pushed the changes back, screenshot has been updated to reflect the design change.